src/funcs/getColumnType.js

```diff
@@ -0,0 +1,23 @@
+// NOTE: possible to use https://www.dbml.org/js-module/#api for the transform to dbml?
+module.exports = function getColumnType(col) {
+  const { data_type: dataType } = col;
+  let columnType;
+  switch (dataType) {
+    case 'character varying':
```

Comment:
should this return columnType = 'varchar'?

Reply:
it does. You can stack cases so you don't have to do multiple return lines, kind of like function overloading.

Reply:
oh nevermind. both character varying and varchar cases are returning 'varchar'...